### PR TITLE
Add installation guide for Multicloud local installation using ansible 

### PR DESCRIPTION
**Pull request by hari-govind**
**What type of PR is this?**

 /kind documentation

**What this PR does / why we need it**:
To add a new section to the documentation for installing local cluster setup of multi-cloud.

**Which issue(s) this PR fixes**:

Fixes [Multi-cloud installation guide should be separate](https://github.com/sodafoundation/multi-cloud/issues/1342)

**Test Report Added?**:
/kind NOT-TESTED

**Test Report**:

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by PravinRanjan10** [comment]
@anvithks @skdwriting Please review this.

File: content/soda-gettingstarted/multicloud-local-installation-using-ansible.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+```bash
+chmod +x ./install_ansible.sh && ./install_ansible.sh
+ansible --version # Ensure Ansible version 2.5.1 for Ubuntu 18.04.
+```
+--- 
+
+### Configure SODA Multi-cloud installer and environment variables
+
+A SODA release conists of various projects which have their own release cycles and versions.
```
Comment (anvithks) [inline content/soda-gettingstarted/multicloud-local-installation-using-ansible.md:97]: conists - > consists

File: content/soda-gettingstarted/multicloud-local-installation-using-ansible.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
+---
+title: Multi-cloud Local Cluster Installation using Ansible
+menuTitle: ""
+disableToc: true
+tags: ["user guide", "multi-cloud", "ansible"]
```
Comment (anvithks) [inline content/soda-gettingstarted/multicloud-local-installation-using-ansible.md:5]: Remove tag 'user guide' and add tag "installation guide"

**Review by anvithks — CHANGES_REQUESTED** [review]
Please check the minor comments.

File: content/soda-gettingstarted/multicloud-local-installation-using-ansible.md (hunk 2 above)
Comment (hari-govind) [reply content/soda-gettingstarted/multicloud-local-installation-using-ansible.md]: Fixed both issues

**Comment by anvithks** [comment]
#SODA-PR-ACCEPTED

**Comment by PravinRanjan10** [comment]
LGTM

**Review by anvithks — APPROVED** [review]
LGTM

**Pull request merged**